### DISCUSSION TITLE
PCAP Retention

### DISCUSSION THREAD

**Discussion by cvfdblaze624** [Security-Onion-Solutions/securityonion#3591] [Unsupported Versions] [answered]
So I have a config question I can't seem to find the answer to, I mounted a larger storage device to store PCAP files under /nsm/pcap but my retention seems to be stuck at 3 weeks.  Both / and /nsm/pcap are both at 6% and 5% use respectively.  I was wondering if I could change something to have it keep PCAP files longer but haven't managed to find where to do that, I saw a post mentioning changing /opt/so/conf/steno/config to increase MaxDirectoryFiles but when I did that it was a temp fix that was overwritten by saltstack I would assume.  Thanks!

**Comment by Masaya-A** [comment]
Change the `MaxDirectoryFiles` value in `/opt/so/saltstack/default/salt/pcap/files/config` and then `so-pcap-restart`.
If you don't want reset the config by `soup`, copy the config file from `/opt/so/saltstack/default/salt/pcap/files/config` to `/opt/so/saltstack/local/salt/pcap/files/config` then edit the local one.

**Reply by cvfdblaze624** [reply]
Thank you so much for the answer, I will test this and see if it corrects the problem

**Comment by dougburks** [comment, marked as answer]
Added to documentation:
https://docs.securityonion.net/en/2.3/stenographer.html#maxdirectoryfiles
